package.json

```diff
@@ -1,50 +1,88 @@
 {
-  "name": "@fabric-ds/css-root",
-  "private": true,
-  "version": "0.0.0",
-  "workspaces": [
-    "packages/*"
+  "name": "@fabric-ds/css",
+  "description": "Fabric CSS core components and utilities",
+  "version": "1.1.1",
+  "files": [
+    "dist"
   ],
+  "style": "dist/fabric.min.css",
+  "exports": {
+    ".": "./dist/fabric.min.css",
+    "./tailwind-config": "./dist/tailwind/tailwind.config.js",
+    "./component-classes": "./dist/component-classes/index.js"
+  },
+  "typesVersions": {
+    "*": {
+      "./component-classes": [
+        "dist/index.d.ts"
+      ]
+    }
+  },
   "scripts": {
-    "build": "NODE_ENV=production lerna run build --stream",
-    "postbuild": "./scripts/reportSizes.mjs",
+    "eik:login": "npx @eik/cli login",
+    "eik:publish": "npx @eik/cli publish",
+    "eik:alias": "eik package-alias @fabric-ds/css",
+    "build": "NODE_ENV=production postcss src/fabric.css --o dist/fabric.min.css && npm run copy && tsc",
+    "copy": "cp -r src/utils/tailwind dist && cp -r src/utils/component-classes dist",
```

Comment:
We don't need to copy this anywhere now that we're just shipping it all as one module, right?

Reply:
Nah, actually probably don't need to since it's all js anyways.

Reply:
It was just for the convenience of only including the `dist` folder in the release and making paths etc a bit simpler, but it's not necessary.